### PR TITLE
Plugin manager: Detail view

### PR DESCRIPTION
**Pull request by koke**
Introduces the plugin detail view, where you can (de)activate a plugin, toggle autoupdates, visit its homepage, and remove it. It's not a complete match to the initial design, but I think it's good enough for a first PR.

See #7471

## Not implemented yet

- Plugin manual update
- Don't show autoupdates for plugins not in the directory
- Plugin directory link
- Changes in the detail view don't show up in the plugin list until it refreshes
- Persistence/caching

## To test:

Verify that you can:

- Activate/deactivate a plugin
- Toggle autoupdates
- Visit a plugin homepage
- Remove a plugin

Needs review: @elibud

### CODE REVIEW

**Comment by elibud** [comment]
@koke this only works correctly with an updated version of Jetpack beta, is that right?

Functionality wise it all checks out, I'm going to look at the code now.

- [x] Activate/deactivate a plugin
- [x] Toggle autoupdates
- [x] Visit a plugin homepage
- [x] Remove a plugin

**Comment by koke** [comment]
Yes, only for Jetpack beta.

File: WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift [hunk 1]
```diff
@@ -0,0 +1,168 @@
...
+                homeLink
+                ]),
+            ImmuTableSection(optionalRows: [
+                removeRow
+                ])
+            ])
+    }
+
+    private func confirmRemovalAlert(plugin: PluginState) -> UIAlertController {
+        let message = NSLocalizedString("Are you sure you want to remove \(plugin.name)?", comment: "Text for the alert to confirm a plugin removal")
```
[the remark this answers is not in the document]
Comment (koke) [reply WordPress/Classes/ViewRelated/Plugins/PluginViewModel.swift]: 😱 somehow I forgot to type the rest of the alert. The design already had that part in it.

screen shot 2017-10-31 at 07 47 32

**Comment by koke** [comment]
Pushed a fix with a better confirmation message. Ready for another look

**Comment by elibud** [comment]
👌 🚢

**Pull request merged**